src/packages/client/src/__tests__/integration/errors/source-map-support/test.ts

```diff
@@ -0,0 +1,18 @@
+import 'source-map-support/register'
+import { getTestClient } from '../../../../utils/getTestClient'
+
+// X is here to have a different stack trace in js and ts
+type X = {}
```

Comment:
I think can only see a snapshot for TS though. Is it interesting to have one for JS too?

Reply:
What I mean with this comment is, that the underlying transpiled JS is different than the TS.
So in pure JS this can't happen.

Reply:
New comment is 💯